### PR TITLE
Fixes #25 - Provide instructions in Alert emails

### LINKED ISSUE #25: Add "Next Steps" to Alert Emails
People receiving alert emails don't necessarily know what to do with them. The email itself should include instructions.

Maybe something similar to the Triage segment here https://docs.telemetry.mozilla.org/tools/alerts.html

### PR DESCRIPTION
**Pull request by chutten**

### CODE REVIEW

**Comment by chutten** [comment]
Not sure who all wants to pile on reviewing this code.

Also not sure when was the last time we deployed medusa. The evolution link from #29 never started being sent.

File: src/clj/medusa/alert.clj [hunk 1]
```diff
@@ -39,12 +39,42 @@
       (let [[earliest-build latest-build] (changesets/bounding-buildids date "mozilla-central") ; get the buildids for the given date
             changeset-url (changesets/pushlog-url earliest-build latest-build "mozilla-central")]
         (log/info "Changeset URL" changeset-url)
-        (send-email (str "Alert for " metric_name " (" detector_name ") on " date)
-                    (str "Alert details: " alert-url
+        (send-email (str "Telemetry Alert for " metric_name " on " date)
+                    (str "We have detected a change in the Telemetry probe " metric_name " in Firefox Nightly builds from " date
```
Comment (erikrose) [inline src/clj/medusa/alert.clj]: Could add a period at the end of this sentence if you liked

File: src/clj/medusa/alert.clj [hunk 2]
```diff
@@ -39,12 +39,42 @@
...
                          "\n\n"
-                         "Evolution dashboard: " evo-url
+                         "Alert details: " alert-url
+                         "\n"
+                         "A list of changes new to Nightly builds on " date ": " changeset-url
```
Comment (erikrose) [inline src/clj/medusa/alert.clj]: Could remove "A list of" for brevity

File: src/clj/medusa/alert.clj [hunk 3]
```diff
@@ -39,12 +39,42 @@
...
+                         "\n"
+                         "The value of " metric_name " over time: " evo-url
```
Comment (erikrose) [inline src/clj/medusa/alert.clj]: So much clearer to a novice like me than "evolution dashboard" :-)

File: src/clj/medusa/alert.clj [hunk 4]
```diff
@@ -39,12 +39,42 @@
...
+                         "\n\n"
+                         "1. File a bug to track your investigation. You can just copy this email into the bug Description to get you started."
+                         "\n"
+                         "2. Reply-All to this email to let the list know that you are investigating. Include the bug number so we can help out."
+                         "\n"
+                         "3. Triage the alert. You can find instructions here: " alert-url
+                         "\n\n"
+                         "If you have any problems, please ask for help on the #telemetry IRC channel or on Slack in #fx-metrics. We'll give you a hand."
+                         "\n\n"
+                         "What is this:"
```
Comment (erikrose) [inline src/clj/medusa/alert.clj]: I'd say "What this is:" or else "What is this?" I had to read it a few times before my brain quit stumbling over it. :-)

**Comment by chutten** [comment]
Took the comments, as they were right and good.

**Comment by chutten** [comment]
@erikrose Do you remember how medusa is deployed? Is it just a matter of rebasing the box and running \*checks notes* `lein cljsbuild once release`?

**Comment by erikrose** [comment]
And then you have to restart it using supervisor:

''sudo lein cljsbuild once release
''sudo supervisorctl
''supervisor> restart medusa

**Comment by chutten** [comment]
I have performed the ritual as described and have hopes that the next batch of alert emails will have the new, improved call to action.

Thanks!

**Pull request merged**